PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -1720,6 +1722,30 @@ intptr_t CALLBACK FindReplaceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARA
 					setChecked(IDC_IN_SELECTION_CHECK, inSelChecked);
 				}
 

```

Comment:
Please add the following comment here:
/*
In the scenario where the user clicks the action button (Count, Find All in Current Document, Replace All, Mark All, or Clear All marks) without activating the Find/Replace dialog, the "In Selection" checkbox could be auto-changed after the button click. To prevent the search from running with this unintended state, the search message has been removed from the queue. Then, launch a message box to alert the user that the search didn't run and they need to verify the settings.
*/